packages/client/src/parse-callback.ts

```diff
@@ -2,17 +2,17 @@ import qs from 'query-string';
 
 export interface AuthenticationResult {
   code?: string;
+  state?: string;
   error?: string;
   error_description?: string;
 }
 
-export const parseRedirectCallback = (url: string) => {
+export const parseRedirectCallback = (url: string): AuthenticationResult => {
```

Comment:
any conventions on the renturn type of method?  @wangsijie @simeng-li @gao-sun

Reply:
both are good for me